src/table/components/head/HeadCellMenu.tsx

```diff
@@ -139,8 +141,37 @@ export default function HeadCellMenu({ column, tabIndex }: HeadCellMenuProps) {
                 ],
               },
             ],
+            [
+              {
+                id: 1,
+                itemTitle: translator.get('SNTable.MenuItem.AdjustColumnSize'),
+                onClick: (evt: React.MouseEvent<HTMLLIElement>) => {
+                  evt.stopPropagation();
+                  evt.preventDefault();
+                  setOpenMenuDropdown(false);
+                  setFocusOnClosetColumnAdjuster(anchorRef);
+                },
+                icon: <ColumnSize />,
+                enabled: true,
+              },
+            ],
           ]
-        : []),
+        : [
```

Comment:
why not just add the object last in the array?

Reply:
Not sure I understand.  `showSearchMenuItem ? []: []`

Reply:
yeah but that then spread into an array so you could do this
```
() => [
      ...(showSearchMenuItem
        ? [
          ...
          ]
        : []),
      {columnWidthItme},
    ],
```

Reply:
very true